CONTRIBUTING.md

```diff
@@ -1,6 +1,28 @@
+Before reporting a bug
```

Comment:
to be consitent with usage elsewhere, I would use the Markdown `#` and `##` syntax in the future